### PR TITLE
Update Wonderland SDK for V3 beta

### PR DESCRIPTION
**Pull request by msub2**
- Updates material handling to work with the latest Wonderland Engine (currently 1.2.0)
- Removes the skips from the Wonderland Engine tests
- Updates test project to use latest version
- Adds WLE beta units

### CODE REVIEW

File: wonderland/src/oldWLE.js [hunk 1]
```diff
@@ -170,8 +174,18 @@ WL.registerComponent(
     },
 
     loadBanner: async function (adUnit, format, style) {
+      const {
+        format: adjustedFormat = format,
+      } = getV3BetaUnitInfo(adUnit);
```
Comment (limbofeather) [inline wonderland/src/oldWLE.js:179]: Same goes here

**Review by limbofeather — APPROVED** [review]
Left a few comments but generally good to merge.

File: tests/wonderland/wonderland-test.wlp [hunk 2]
```diff
@@ -993,11 +993,12 @@
                     "type": "zesty-banner",
                     "zesty-banner": {
                         "adUnit": "999999",
-                        "beacon": true,
+                        "beacon": false,
```
[the remark this answers is not in the document]
Comment (msub2) [reply tests/wonderland/wonderland-test.wlp:996]: We aren't currently capturing the beacon during tests and this hits our real endpoint at the moment.

**Comment by msub2** [comment]
I understand the concern around getV3BetaUnitInfo, I'll merge this PR now and then I can fix it for Wonderland and A-Frame together in a follow up

**Pull request merged**